packages/components/src/components/CheckboxGroup/CheckboxGroup.test.tsx

```diff
@@ -0,0 +1,123 @@
+import 'react-native'
+import { fireEvent, render, screen } from '@testing-library/react-native'
+import React from 'react'
+
+import { CheckboxGroup } from './CheckboxGroup'
+
+const mockedColorScheme = jest.fn()
+
+jest.mock('react-native/Libraries/Utilities/useColorScheme', () => {
+  return {
+    default: mockedColorScheme,
+  }
+})
+
+describe('CheckboxGroup', () => {
+  const onSelectionChangeSpy = jest.fn()
+
+  const items = [
```

Comment:
One item should validate non-object, just string list item and corresponding behavior aligns as expected (text/a11y set to string, everything else not set).

Reply:
Added additional tests to test array of strings